frontend/src/services/localStorage.ts

```diff
@@ -0,0 +1,7 @@
+export const saveUserToStorage = (user = {}) => {
+  localStorage.setItem('loginedUser', JSON.stringify(user));
+}
+
+export const getUserFromStorage = () => {
+  return JSON.parse(localStorage.getItem('loginedUser')!)
```

Comment:
the ! mark is not really good idea to use here - since this just hides the potential issue with null